### PR TITLE
Fix typo in docs

### PR DESCRIPTION
**Pull request by pugnascotia**
Some hyphens were swapped for question marks in bf4c364. Swap them back.

### CODE REVIEW

File: docs/reference/docs/index_.asciidoc [hunk 1]
```diff
@@ -235,7 +235,7 @@ and a status code of 409. See <<optimistic-concurrency-control>> for more detail
 [[index-routing]]
 ===== Routing
 
-By default, shard placement ? or `routing` ? is controlled by using a
+By default, shard placement - or `routing` - is controlled by using a
```
Comment (jrodewig) [inline docs/reference/docs/index_.asciidoc:238]: Em dashes are the preferred punctuation here.
```suggestion
By default, shard placement -- or `routing` -- is controlled by using a
```

**Review by jrodewig — APPROVED** [review]
LGTM. Made a minor change to replace the hyphens with em dashes.

THanks for fixing this @pugnascotia.

**Comment by pugnascotia** [comment]
Ah, I did wonder about the hyphens. Thanks for the fix.

**Comment by pugnascotia** [comment]
Backports:

`7.x`: fd91617b190
`7.7`: 9d0f2e07d73

**Pull request merged**